packages/dapp/src/index.js

```diff
@@ -3,18 +3,20 @@ import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
 import {Provider} from 'wagmi';
-import {connectors, provider, connectorStorageKey} from './web3/connect';
+import {connectors, provider, connectorStorageKey} from './utils/connect';
+import { BrowserRouter } from "react-router-dom";
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
+
 root.render(
-  <React.StrictMode>
```

Comment:
wasn't sure if this is super important, I haven't used it before though. but happy to keep it if required

Reply:
@thestrabusiness do you have any thoughts on this?